src/main/java/banking/database/DBConnector.java

```diff
@@ -0,0 +1,38 @@
+package banking.database;
+
+
+import org.sqlite.SQLiteDataSource;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DBConnector {
```

Comment:
Can you use a more meaningful name, e.g. `DBManager`?

Reply:
fixed